src/app/client_sdk/js_util.ml

```diff
@@ -23,6 +23,41 @@ type payload_common_js =
   ; memo : string_js Js.prop >
   Js.t
 
+type payload_party_js =
+  < fee : string_js Js.prop
+  ; feePayer : string_js Js.prop
+  ; nonce : string_js Js.prop
+  ; memo : string_js Js.prop >
+  Js.t
+
+let party_payload_of_js (payload_party_js : payload_party_js) =
+  let fee_payer_pk =
+    payload_party_js##.feePayer
+    |> Js.to_string |> Signature_lib.Public_key.of_base58_check_decompress_exn
+  in
+  let fee_js = payload_party_js##.fee in
+  let fee = Js.to_string fee_js |> Currency.Fee.of_string in
+  let nonce_js = payload_party_js##.nonce in
+  let nonce = Js.to_string nonce_js |> Mina_numbers.Account_nonce.of_string in
+  { Party.Fee_payer.data =
```

Comment:
I'm not sure if these details for constructing a fee payer are 100% right, mind taking a look? @bkase

Reply:
Looks right to me, but maybe @deepthiskumar or @mrmr1993 should also check as I'm not as familiar with the parties logic

Reply:
LGTM -- the missing authorization is added in signParty